plugins/package-managers/cargo/src/main/kotlin/Cargo.kt

```diff
@@ -209,6 +215,10 @@ private fun CargoMetadata.Package.toPackage(hashes: Map<String, String>): Packag
     // https://github.com/rust-lang/cargo/pull/4920
     val declaredLicensesProcessed = DeclaredLicenseProcessor.process(declaredLicenses, operator = SpdxOperator.OR)
 
+    val vcs = repository?.let { VcsHost.parseUrl(it) }
+        ?: getLocalPath()?.let { PackageManager.processProjectVcs(it) }
```

Comment:
This is using the result of `processProjectVcs()` as (unprocessed) `vcs`. 
Just in recent discussion, IIUC, this has been seen undesired. What do you think?
(Looking at other package managers, many places used `VcsInfo.EMPY`)

I'm not against this change, but believe this needs to be clarified first.

Reply:
I also recalled that discussion when writing this code and thought about it. The problem here is that without this code, a package that actually is a project outside of the working tree would not have *any* VCS information at all, neither processed nor unprocessed. Adding this code was the easiest way to fix that, even if strictly speaking the "bonus" of getting the unprocessed VCS non-empty for "real" packages would not be required.

Reply:
> The problem here is that without this code, a package that actually is a project outside of the working tree would not have any VCS information at all, 

I do not understand this, can you help? (I wonder why not `vcs ? VcsInfo.EMPTY; vcsProcesses=...` is possible)

Reply:
> I wonder why not `vcs ? VcsInfo.EMPTY; vcsProcessesed=...` is possible

Ah, I think I now got what you mean: Basically, use `processProjectVcs()` only for `vcsProcessed`. That I could do.

Reply:
Should have been `=` instead of `?`, but you got it. 

You don't have to change it. I just wanted to have it clear, since in my NPMSupport improvements PR we discarded the changes for `vcs` unprocessed recently. And I believe it was because of rationale that it shouldn't be processed. I'm also fine with changing the decision and just keep your code as-is in this regard. up to you.

Reply:
I've already changed it locally, and it reduces the diff in test output, so that's probably good thing for review readability anyway.